com.unity.shadergraph/Editor/Importers/ShaderGraphImporter.cs

```diff
@@ -348,12 +348,12 @@ static ShaderGraphVfxAsset GenerateVfxShaderGraphAsset(GraphData graph)
             // we need to override graph.isSubgraph, so save old state to restore it
             // (this is not great, but whole VFX pipeline is rather hacky at the moment)
             // use try/finally to ensure it always gets restored
+            // UPDATE(2/24/2021) @gabriel.delacruz: Not required anymore, as we are not
+            // forcing graph.isSubgraph to be true. Kept in case we have to go back to that,
+            // but most likely this code will be soon replaced by direct generation from SG
             bool oldIsSubGraph = graph.isSubGraph;
```

Comment:
Caching this does not look needed anymore?

Reply:
I removed the caching and the comment, it is more readable now